docs/site/BelongsTo-relation.md

```diff
@@ -43,6 +43,11 @@ related routes, you need to perform the following steps:
 This section describes how to define a `belongsTo` relation at the model level
 using the `@belongsTo` decorator to define the constraining foreign key.
 
+LB4 also provides an CLI tool `lb4 relation` to generate `belongsTo` relation
```

Comment:
So that the reader doesn't have to come back to this section, can you move it to after the `Relation Metadata` section?

Reply:
I was thinking where is the best place to put it. I don't think after relation metadata is a good idea because we introduce how users can customize keyFrom, keyTo, relation name, and even column name, and most of them are not supported by the CLI. That's why I put it here, right after we introduce the basic idea of each relation.